### PR TITLE
Update images

### PR DESCRIPTION
**Pull request by rhpvorderman**
### Checklist
- [x] Pull request details were added to CHANGELOG.md
- [x] `parameter_meta` for each task is up to date.

### CODE REVIEW

File: bcftools.wdl [hunk 1]
```diff
@@ -48,9 +57,11 @@ task Bcf2Vcf {
     }
 
     parameter_meta {
-        bcf: {description: "The generated BCF from an SV caller", category: "required"}
+        inputFile: {description: "A vcf or bcf file", category: "required"}
```
Comment (JasperBoom) [inline bcftools.wdl:60]:
```suggestion
        inputFile: {description: "A vcf or bcf file.", category: "required"}
```

File: bcftools.wdl [hunk 2]
```diff
@@ -22,23 +22,32 @@ version 1.0
...
         set -e
         mkdir -p "$(dirname ~{outputPath})"
-        bcftools view ~{bcf} -O v -o ~{outputPath}
+        bcftools view \
+        -o ~{outputPath} \
+        -O ~{outputType} \
+        -l ~{compressionLevel} \
+        ~{inputFile}
+        bcftools index --tbi ~{outputPath}
+
```
Comment (JasperBoom) [inline bcftools.wdl:45]:
```suggestion
```

File: htseq.wdl [hunk 3]
```diff
@@ -25,38 +25,39 @@ task HTSeqCount {
...
         Int timeMinutes = 10 + ceil(size(inputBams, "G") * 60)
-        String dockerImage = "quay.io/biocontainers/htseq:0.11.2--py37h637b7d7_1"
+        String dockerImage = "quay.io/biocontainers/htseq:0.12.4--py37hb3f55d8_0"
     }
 
     command {
         set -e
         mkdir -p "$(dirname ~{outputTable})"
         htseq-count \
-        -f ~{format} \
```
Comment (JasperBoom) [inline htseq.wdl:44]: Should this change also be noted in the CHANGELOG?

Comment (rhpvorderman) [reply htseq.wdl:44]: Yes it should.

**Pull request merged**